### PR TITLE
My development cryptic

### PR DESCRIPTION
**Pull request by DPSCryptic**
dont look at:
- i dont even know what i changed
- delete hatchet
- delete tree
- delete woodcutting

did these without looking in skills.core

added a wealth command for those who want it.

added sell all option in shops, this will be lowering the load of the server too

### CODE REVIEW

File: 2006Redone Server/src/redone/game/shops/ShopAssistant.java [hunk 1]
```diff
@@ -367,29 +369,27 @@ public boolean sellItem(int itemID, int fromSlot, int amount) {
...
+					GameLogger.writeLog(player.playerName, "shopselling", player.playerName + " sold " + itemName + " to store id: " + player.myShopId + " for" + GameLogger.formatCurrency(TotPrice2) + " coins");
+					player.getActionSender().sendMessage("You sold " + amount + " " +itemName + " for " + TotPrice2 + " gp." );
 				}
+			} else {
+				player.getActionSender().sendMessage("You don't have enough space in your inventory.");
 			}
 			player.getItemAssistant().resetItems(3823);
 			resetShop(player.myShopId);
 			updatePlayerShop();
+			player.getActionSender().sendMessage("You sold " + amount + " " +itemName + " for " + TotPrice2 + " gp." );
```
Comment (RedSparr0w) [inline 2006Redone Server/src/redone/game/shops/ShopAssistant.java]: It looks like this message is being sent twice?
Already sent on [line#384](https://github.com/dginovker/2006rebotted/pull/61/files#diff-21cdeee7633cd5195bd70dd70768ed7fR384)

File: 2006Redone Server/src/redone/integrations/discord/commands/Commands.java [hunk 2]
```diff
@@ -9,7 +9,7 @@
     public void onMessageCreate(MessageCreateEvent event) {
         Message message = event.getMessage();
         if (message.getContent().equalsIgnoreCase("::commands")) {
-            event.getChannel().sendMessage("```fix`
+            event.getChannel().sendMessage("```fix```"
```
Comment (RedSparr0w) [inline 2006Redone Server/src/redone/integrations/discord/commands/Commands.java]:
```suggestion
            event.getChannel().sendMessage("```fix"
```

File: 2006Redone Server/src/redone/net/packets/impl/Bank10.java [hunk 3]
```diff
@@ -28,14 +28,15 @@ public void processPacket(Client player, int packetType, int packetSize) {
 			player.getPlayerAssistant().useOperate(removeId);
 			break;
 		case 3900:
-			player.getShopAssistant().buyItem(removeId, removeSlot, 5);
+			player.getShopAssistant().buyItem(removeId, removeSlot, 10);
```
Comment (RedSparr0w) [inline 2006Redone Server/src/redone/net/packets/impl/Bank10.java]: Is there any reason to change the 5 option to 10?
I feel like this is just confusing the menu even more.

File: 2006Redone Server/src/redone/net/packets/impl/Bank10.java [hunk 4]
```diff
@@ -28,14 +28,15 @@ public void processPacket(Client player, int packetType, int packetSize) {
...
-			player.getShopAssistant().buyItem(removeId, removeSlot, 5);
+			player.getShopAssistant().buyItem(removeId, removeSlot, 10);
 			break;
 
 		case 3823:
 			if(!player.getItemAssistant().playerHasItem(removeId)) {
 				return;
 			}
-			player.getShopAssistant().sellItem(removeId, removeSlot, 5);
+			player.getShopAssistant().sellItem(removeId, removeSlot, 10);
```
Comment (RedSparr0w) [inline 2006Redone Server/src/redone/net/packets/impl/Bank10.java]: Is there any reason to change the 5 option to 10?
I feel like this is just confusing the menu even more.

File: 2006Redone Server/src/redone/game/shops/ShopAssistant.java (hunk 1 above)
Comment (DPSCryptic) [reply 2006Redone Server/src/redone/game/shops/ShopAssistant.java]: hmm..

it only shows up once, thats why i missed it probably

File: 2006Redone Server/src/redone/net/packets/impl/Bank10.java [hunk 5]
```diff
@@ -28,14 +28,15 @@ public void processPacket(Client player, int packetType, int packetSize) {
...
+			player.getItemAssistant().resetItems(3823);
```
Comment (RedSparr0w) [inline 2006Redone Server/src/redone/net/packets/impl/Bank10.java]: Doesn't look like this line is needed?

Comment (DPSCryptic) [reply 2006Redone Server/src/redone/net/packets/impl/Bank10.java]: Indeed the resetItems is not needed, i should have tested it before. 
The bank method used it and first it was not working (check gifs in discord) so i added it.

File: 2006Redone Server/src/redone/net/packets/impl/Bank10.java (hunk 4 above)
Comment (DPSCryptic) [reply 2006Redone Server/src/redone/net/packets/impl/Bank10.java]: no reason besides the sell 10 option being replaced by sell all

File: 2006Redone Server/src/redone/integrations/discord/commands/Commands.java (hunk 2 above)
Comment (DPSCryptic) [reply 2006Redone Server/src/redone/integrations/discord/commands/Commands.java]: ohh, it gives me an error tho :/

File: 2006Redone Server/src/redone/net/packets/impl/Bank10.java (hunk 4 above)
Comment (RedSparr0w) [reply 2006Redone Server/src/redone/net/packets/impl/Bank10.java]: Probably better to leave it as `5`?
```suggestion
			player.getShopAssistant().sellItem(removeId, removeSlot, 5);
```

File: 2006Redone Server/src/redone/integrations/discord/commands/Commands.java (hunk 2 above)
Comment (DPSCryptic) [reply 2006Redone Server/src/redone/integrations/discord/commands/Commands.java]: ohhh the ``` sorry thought you deleted the " too

File: 2006Redone Server/src/redone/net/packets/impl/Bank10.java (hunk 4 above)
Comment (DPSCryptic) [reply 2006Redone Server/src/redone/net/packets/impl/Bank10.java]: yeah maybe it is better

**Pull request merged**